extension/background/content.js

```diff
@@ -52,9 +52,17 @@ export async function lazyInject(tabId, scripts) {
   ]
     .concat(scripts)
     .concat(["/content/responder.js"]);
+
   for (const script of scripts) {
-    await browser.tabs.executeScript(tabId, { file: script });
+    try {
+      await browser.tabs.executeScript(tabId, { file: script });
+    } catch (error) {
+      const e = new Error("That does not work on this kind of page");
+      e.displayMessage = "That does not work on this kind of page";
+      throw e;
+    }
```

Comment:
This will catch everything, which we don't want. We only want to catch the missing host permission.

So this should be something like:

```js
} catch (error) {
  if (error.message.includes("missing host permission")) { // Not sure if this is the right string
    const e = ...
  } else {
    throw error;
  }
}
```